sys/arm64/arm64/trap.c

```diff
@@ -224,6 +226,38 @@ cap_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
 	userret(td, frame);
 }
 
+static void
+cap_mmu_abort(struct thread *td, struct trapframe *frame, uint64_t esr,
```

Comment:
This is going to be short-lived, right, and be folded into data_abort like on RISC-V?

Reply:
I guess I will look at folding it in now, sure.